indico/modules/events/timetable/controllers/legacy.py

```diff
@@ -206,3 +212,69 @@ def _process(self):
         with track_time_changes():
             fit_session_block_entry(self.session_block.timetable_entry)
         return jsonify_data(flash=False)
+
+
+class RHLegacyTimetableEntryMove(RHManageTimetableEntryBase):
+    """Moves a TimetableEntry into a Session or top-level timetable"""
+
+    def _process_POST(self):
+        current_day = dateutil.parser.parse(request.form.get('day')).date()
+        return jsonify_template('events/timetable/move_entry.html', event=self.event_new,
+                                top_level_entries=self._get_session_timetable_entries(),
+                                current_day=current_day, timetable_entry=self.entry,
+                                parent_entry=self.entry.parent)
+
+    def _process_PATCH(self):
```

Comment:
But I need to differentiate somehow between request to show a dialog and request to move the entry.